src/value_generators/test/DocumentGeneratorTestCases.yml

```diff
@@ -606,15 +606,10 @@ Tests:
     ThenReturns:
     - { "int" : { "$numberLong" : "2000000000"}}
 
-  # Now moves with time so would always fail until PERF-2086 is available.
-#  - Name: Now
-#    GivenTemplate:
-#      date: {^Now: {}}
-#    ThenReturns:
-#    # The $date notation is extended json
-#    # (see https://docs.mongodb.com/manual/reference/mongodb-extended-json/#bson.Date).
-#    # Check with linux command line `date --utc -d@$((1453689676330 / 1000.0))  +"%Y-%m-%dT%H:%M:%S.%3N"`
-#    - { "date" : { "$date" : { "$numberLong" : "1453689676330" } } }
```

Comment:
Should we still leave a comment referencing PERF-2086? Since that ticket is on the backlog product perf might want to update this again if they get the chance to do that ticket.

Reply:
_ThenExecuteAndIgnore_ fixes this issue for me. I would be happy to just close PERF-2086 out (it is a nice to have IMO but a very small edge case).